src/theme/components/card.ts

```diff
@@ -133,60 +133,69 @@ const ReservoirCard = defineMultiStyleConfig({
       return {
         base: {
           containerType: "inline-size",
-          "@container (min-width: 0px)": {
-            "[data-wrapper]": {
-              flexFlow: isRow ? "column nowrap" : null,
-            },
-            "[data-actions]": {
-              flexShrink: isAlignedRightActions ? "0" : null,
-              marginStart: "0",
-              marginTop: "xs",
-              maxWidth: "100%",
-            },
-            "[data-body]": {
-              width: "100%",
-            },
-            ...(isRow && {
-              "[data-image]": {
+          ...setContainerStyles({
+            breakpoint: "base",
+            styles: {
+              "[data-cardWrapper]": {
+                flexFlow: isRow ? "column nowrap" : null,
+              },
+              "[data-actions]": {
+                flexShrink: isAlignedRightActions ? "0" : null,
+                marginStart: "0",
+                marginTop: "xs",
                 maxWidth: "100%",
-                margin: imageIsAtEnd ? "var(--nypl-space-m) 0 0" : null,
+              },
+              "[data-body]": {
                 width: "100%",
-                ...imageSizesMinWidth0[imageSize],
               },
-            }),
-          },
-          [`@container (min-width: ${breakpoints.sm})`]: {
-            "[data-body]": {
-              flexBasis: isRow ? "100%" : null,
+              ...(isRow && {
+                "[data-imageWrapper]": {
+                  maxWidth: "100%",
+                  margin: imageIsAtEnd ? "var(--nypl-space-m) 0 0" : null,
+                  width: "100%",
+                  ...imageSizesMinWidth0[imageSize],
+                },
+              }),
             },
-          },
-          [`@container (min-width: ${breakpoints.md})`]: {
-            "[data-wrapper]": {
-              flexFlow: isRow ? "row" : null,
-            },
-            "[data-actions]": {
-              flexShrink: "0",
-              marginStart: "m",
-              marginTop: "0",
-              maxWidth: "180px",
-            },
-            "[data-body]": {
-              display: "block",
-              flexFlow: "row nowrap",
-              width: "auto",
+          }),
+          ...setContainerStyles({
+            breakpoint: "sm",
```

Comment:
@jackiequach and @EdwinGuzman – During dev, I would be more comfortable using the semantic breakpoint names here (`smallMobile`, `tablet`, etc.). Do you have a preference one way or the other?

Reply:
I think once the breakpoints PR is integregated the semantic names would be used. For now it can use the current breakpoint keys.